.eslintignore

```diff
@@ -10,6 +10,10 @@ lerna-debug.log
 .DS_Store
 .vscode/
 
+# `/bundles` is used during the release step as the checkout location for the
```

Comment:
This file gets auto-generated by the `ignore-sync` script, so you only need to put it in the `.gitignore` file and run `npm run ignore-sync`. (See https://github.com/foray1010/ignore-sync#readme)

Reply:
Yeah, these are the changes that script made. These files aren't ignored, so I assume we want them checked in?

Reply:
Oh I see, sorry I think the comments made me think they were manually checked copied to each place. NM.

Reply:
OIC, yeah I was surprised it copied the comments also.